department-of-veterans-affairs/caseflow
# Shalliburton/caseflow 1404 time slot tooltip

Resolves [CASEFLOW-1404](https://vajira.max.gov/browse/CASEFLOW-1404)

### Description

Adds a tooltip to the POA name on the time slot details and allows the hover events to pass down so that the docket type badge also has the tooltip available

### Acceptance Criteria
- [ ] Display the full POA name in a tooltip when hovering over a truncated name
- [ ] Display the docket type in a tooltip when hovering over the [L] or [H] badges

### Testing Plan

1. Shadow user `BVASYELLOW`: http://localhost:3000/test/users
2. Navigate to the hearing schedule: http://localhost:3000/hearings/schedule
3. Click the “Schedule Veteran” button at the top left
4. Select the St. Petersburg, FL RO from the dropdown
5. Click the “AMA Veterans Waiting” tab and click the name of one of the Veterans
6. On the Case Details page, click the “Schedule Veteran” option from the task list
7. Choose a virtual hearing day form the list to show the time slots
8. Choose one of the time slots to schedule the veteran
9. Repeat the steps above to get another veteran to schedule
10. Once back on the case details page with the virtual hearing day selected, hover over the docket type badge and the poa name and ensure that you see the tooltip. Additionally, tab to the fields and ensure that the focus indicator wraps the content correctly
